### ISSUE TITLE
New  request: CIA Freedom of Information Act Electronic Reading Room

### ISSUE THREAD

**Issue by Rexadev** [openzim/zim-requests#593]
*Please use the following format for a ZIM creation request (and delete unnecessary information)*

- Website URL: https://www.cia.gov/readingroom/
- License: 
- Desired ZIM Title: CIA Freedom of Information Act Electronic Reading Room
- Desired ZIM Description: 
- Desired ZIM Icon –png (URL or attach one): https://upload.wikimedia.org/wikipedia/commons/thumb/2/25/Seal_of_the_Central_Intelligence_Agency.svg/200px-Seal_of_the_Central_Intelligence_Agency.svg.png
- Language (ISO 639-3): **eng**
- Is this a MediaWiki?: **no**

**Comment by RavanJAltaie** [comment]
@Popolechien The URL is more like a 1 page of information, does it worth creating a zim file for it?

**Comment by Rexadev** [comment]
@RavanJAltaie How about https://www.cia.gov/index.html

**Comment by Popolechien** [comment]
@RavanJAltaie Yeah it's a database that needs to be queried, not something we can handle out of the box. 

@Rexadev We already have their world factbook, I don't think there's much else that the generic cia.gov website can give us as "educational material", except maybe the Langley Files Podcast. @RavanJAltaie can you please try this subsite? https://www.cia.gov/stories/story/the-langley-files/

**Comment by RavanJAltaie** [comment]
@Popolechien recipe created
https://farm.openzim.org/recipes/langelyfiles_en
Will post the library file link once successful & ready

**Comment by Rexadev** [comment]
@RavanJAltaie File not in library and recipe page is 404
